webpack.dev.js

```diff
@@ -6,7 +6,7 @@ const express = require('express');
 
 module.exports = merge(common, {
     mode: 'development',
-    devtool: 'eval-source-map',
+    devtool: 'source-map',
```

Comment:
Hva er forskjellen på `eval-source-map` og `source-map`?

Reply:
skal visst egentlig være eval-source-map, den er mer nøyaktig, men ikke egnet for prodbruk. source-map kan derimot brukes i prod :)